snorkel/parser.py

```diff
@@ -1,51 +1,55 @@
 # -*- coding: utf-8 -*-
 import atexit
-from bs4 import BeautifulSoup
 import codecs
-from collections import defaultdict
 import glob
 import json
-import lxml.etree as et
 import os
 import re
-import requests
 import signal
-from subprocess import Popen
 import sys
 import warnings
+import requests
+import lxml.etree as et
+
+try:
+    import spacy
+except:
+    print>>sys.stderr,"Warning, unable to load 'spaCy' module"
```

Comment:
Presumably this is for a subclass of `Parser`... should reorg by making a `parsers` dir and putting `Parser` subclasses there?